scripts/abstract_task.py

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env python
+import rospy
+
+class AbstractTask:
+
+    # Abstract method
+    def get_preferred_velocity(self):
```

Comment:
Passing in a timestamp means that the task has to be prepared for a reasonable range of timestamps to be entered or return an error. I'd rather have the relationship implicitly be the current time.